olp-cpp-sdk-core/src/http/android/HttpClient.java

```diff
@@ -51,7 +52,8 @@
   public static final int TIMEOUT_ERROR = -7;
   public static final int THREAD_POOL_SIZE = 8;
 
-  private int uniqueId = (int) (System.currentTimeMillis() % 10000);
+  private static AtomicInteger uniqueIdCounter = new AtomicInteger(0);
+  private int uniqueId = -1;
```

Comment:
Currently, requestId 0 is considered invalid on cpp side, is it ok here?

Reply:
It is not request id here, it is the id of NetworkAndroid object. 

But I can change that in order to align it with other places in code,